package.json

```diff
@@ -15,8 +15,8 @@
     "clean": "npx rimraf dist",
     "test": "jest --detectOpenHandles --forceExit",
     "test:integration": "npm run test -- --testPathPattern=test/integration",
-    "lint": "prettier --check \"**/*.{ts,tsx}\"  && eslint .",
-    "format": "prettier --write \"**/*.{ts,tsx}\" && eslint . --fix"
+    "lint": "prettier --check \"**/*.{ts,tsx}\" --ignore-path \".gitignore\" && eslint .",
```

Comment:
I guess we can ignore it with a .prettierignore?

Reply:
I'd say as long as one file can do the job for both (git + prettier) it's nicer that way?

Reply:
Sure, no worries. This is fine for now.